app/services/create_proposed_organisation_edit.rb

```diff
@@ -34,8 +35,8 @@ def merge_in_non_published_fields
     end
   end
 
-  def send_email_to_superadmin_about_org_edit
-    superadmin_emails = user_klass.superadmins.pluck(:email)
-    mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails).deliver_now
-  end
+  # def send_email_to_superadmin_about_org_edit
+  #   superadmin_emails = user_klass.superadmins.pluck(:email)
+  #   mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails).deliver_now
+  # end
```

Comment:
thanks @decareano - now I look at it I can see a bigger refactoring to simplify the whole method, to make the code look like this stateless service I created in WSO:

```rb
class UpgradeUserToPremium

  def self.with(user, time, stripe_id, payment_source_klass = PaymentSource::Stripe, subscription_klass = Premium)
    payment_source = payment_source_klass.new(identifier: stripe_id)
    user.subscription = subscription_klass.new(started_at: time, payment_source: payment_source)
    user.save
  end

end
```
but that could be the subject of a completely different ticket/PR